api/core/storage/slack/slack_types.py

```diff
@@ -87,19 +89,17 @@ class SlackEventData(NamedTuple):
     blocks: list[dict[str, Any]] | None = None
 
 
-class SlackAuthorization(NamedTuple):
-    enterprise_id: str | None
+class SlackAuthorization(BaseModel):
     team_id: str
     user_id: str
     is_bot: bool
     is_enterprise_install: bool
+    enterprise_id: str | None = None
 
 
-class SlackWebhookEvent(NamedTuple):
+class SlackWebhookEvent(BaseModel):
```

Comment:
@guillaq the nested parts of the SlackWebhookEvent are parsed as dict instead of NamedTuple, which breaks the code so I'll stick to BaseModel for now.

Reply:
done in https://github.com/WorkflowAI/WorkflowAI/pull/176/commits/e69a20d7b23e909eaf380e9a569deadf2fcce8b5